index.html

```diff
@@ -1,277 +1,334 @@
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@500&family=Poppins&family=Roboto:wght@700&display=swap"
+    rel="stylesheet" />
+</head>
+
+<body id="body">
+
+  <!--------- header start----------->
+  <main id="main">
     <header>
-      <nav class="navbar">
-        <img src="images/Logo Placeholder.png" alt="logo" />
-        <a class="toggler" target="_self" href="nav-mobile.html">
-          <img src="images/Icon - Menu.png" alt="icon image" />
-        </a>
+      <nav class="navbar-container">
+        <div class="logo-container">
+          <img class="branding-log" src="#" alt="Sazzad" t />
+          <img id="hamImg" src="./icons/Enabled.svg" alt="hamburger menu" />
+        </div>
+        <div class="mobile-menu">
+          <img id="crossImg" src="./icons/cross.png" alt="menu toggler link" />
+          <ul>
+            <li>
+              <a class="mob-items" href="#portfolio" target="_self" rel="noopener">Portfolio</a>
+            </li>
+            <hr>
+            <li>
+              <a class="mob-items" href="#about" target="_self" rel="noopener">About</a>
+            </li>
+            <hr>
+            <li>
+              <a class="mob-items" href="#form" target="_self" rel="noopener">Contact</a>
+            </li>
+            <hr>
+          </ul>
+        </div>
+        <ul class="desk-menu">
+          <li>
+            <a href="index.html" target="_self" rel="noopener">Portfolio</a>
+          </li>
+          <li>
+            <a href="#about" target="_self" rel="noopener">About</a>
+          </li>
+          <li>
+            <a href="#form" target="_self" rel="noopener">Contact</a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/abdus-sattar-a41a26215/" target="_blank" rel="noopener">
+              <img src="icons/Mail.png" alt="message link">
+            </a>
+          </li>
+        </ul>
       </nav>
     </header>
-    <!--Main part start-->
-    <section class="main">
-      <article class="main-article">
-        <h2>Hey There.
-            I’m Sazzad</h2>
-        <h3>I’am a Software Developer</h3>
-        <p>I can help you builds a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-            don’t hestiate to contact me.</p>
-      </article>
-      <div class="social-media">
+    <!-- headline-container start -->
+    <section class="section">
+      <div class="headline-container">
+        <div class="headline-text">
+          <h1>Hey There,<br>
+            I'm Sazzad
+          </h1>
+          <h2>I'm a Software Developer</h2>
+          <article class="artical">
+            I can help you build a product, feature or website Look through some of my work and experience! If you like
+            what you
+            see and have a project you need coded,
+            don't hestiate to contact me.
+          </article>
```

Comment:
I think while converting the design, you forgot to change the font sizes. Kindly look at the fonts of the following elements.
![image](https://user-images.githubusercontent.com/61361037/221350008-d09ad0ec-fb7e-46a2-a81e-2a1852a9b731.png)
Since the screen size on desktop is larger, so is the text of all these items. Kindly update the sizes.